packages/licensing/phpunit.xml.dist

```diff
@@ -10,7 +10,7 @@
 			<exclude>
 				<directory suffix=".php">tests</directory>
 				<directory suffix=".php">vendor</directory>
-				<directory suffix=".php">wordpress</directory>
+				<directory>wordpress</directory>
```

Comment:
Why do the other two specify a suffix, but this one doesn't?

Reply:
My initial thought was that `wordpress` folder also includes js files. But apparently, js files would not affect PHP coverage results. I'll add these suffixes back :)

Reply:
added suffixes back in 03aa249

Reply:
Ok.

Personally I might have gone the other way, just excluding the whole directories instead of just certain suffixes in case it can optimize the exclusion. But absent actually determining that it's an optimization, I'm fine with what you did.